odoo/orm/table_objects.py

```diff
@@ -98,14 +102,15 @@ def __init__(
         if message:
             self.message = message
 
-    @property
-    def definition(self):
+    def get_definition(self, pool: Registry):
```

Comment:
pool is quite a bad name
```suggestion
    def get_definition(self, registry: Registry):
```

Reply:
Changed.